opensearch-search-quality-evaluation-plugin/scripts/create-judgments-now.sh

```diff
@@ -18,4 +18,4 @@ echo "Creating judgments index..."
 #                                              }'
 
 echo "Creating judgments..."
-curl -s -X POST "http://localhost:9200/_plugins/search_quality_eval/judgments?click_model=coec&max_rank=1"
+curl -s -X POST "http://localhost:9200/_plugins/search_quality_eval/judgments?click_model=coec&max_rank=20"
```

Comment:
so this is to go a lot deeper....?

Reply:
Yes, 20 was the original. 1 was for testing.